constants.py

```diff
@@ -55,6 +55,20 @@ def remove_comments(text):
     return re.sub(r'  //.*\n', r'', text)
 
 
+def create_filepath_with_prefix(filename_prefix, filename):
```

Comment:
Why is this function in `constants`? I think this might be more relevant for `fs_services`,  wdyt?

Reply:
I agree, but I also don't think this function is needed. Let's use an approach similar to the one that @srikanthkb is taking here (but see my follow-up comment too).